README.md

```diff
@@ -1,2 +1,6 @@
 # code_style
 Code styles, and tools for different languages
+
+# General conventions
+
+- Mobile Clients will send a header `bv-app-version` with the app version for each api request.
```

Comment:
so.. `platform/typeApp/version/build` where:
`platform` : platform who make the request in **lower case** (ej: `ios`, `android`)
`typeApp` :  app profile who make the request in **lower case** (ej: `customer`, `driver`, `admin`) if there are only one app could be `mobile` for mobile apps
`version` : current app version following semver standard (`x.y.z`)
`build` : current build version, could be empty, number or letters 

am i right?